client/src-worker/turbo-resolver/resolver.js

```diff
@@ -170,7 +170,8 @@ export class Resolver {
             parentNode: 'root'
           })
         });
-      });
+      })
+      .catch(reject);
```

Comment:
I think I understand the missing rejection.
Could you also help to add coverage in `client/test-worker/turbo-resolver/resolver.spec.js` on handling rejected `registry.batchFetch`?

Reply:
Done, I also changed npm-http to get the real error message (this will also be useful in the case of authentication).